modules/multi-architecture-scheduling-examples.adoc

```diff
@@ -0,0 +1,139 @@
+
+There are various cases to consider when you are scheduling workloads on a cluster with compute nodes of different architectures. 
+
+Using node affinity to schedule workloads on a node:: To allow a workload to be scheduled on only a set of nodes with architectures supported by its images, you can set the `spec.affinity.nodeAffinity` field in your pod template specification.
++
+.Example deployment with the `nodeAffinity` set to certain architectures 
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata: [...]
+spec:
+  [...]
+  template:
+    [...]
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: kubernetes.io/arch
+                operator: In
+                values: <1>
+                - amd64
+                - arm64
+----
+<1> Specify the set of supported architecture. Valid values include `amd64` and `arm64`.
+
+Tainting every node for a specific architecture:: You can taint a node to avoid workloads that are not compatible with its architecture to be scheduled on that node. In the case that your cluster is using a `MachineSet` object, you can add parameters to a `.spec.template.spec.taints` field to avoid workloads being scheduled on nodes with non-supported architectures.
+
+* Before you can taint a node, you must scale down the `MachineSet` object or remove available machines. You can scale down the machine set with the following commands: 
++
+[source,terminal]
+----
+$ oc scale --replicas=0 machineset <machineset> -n openshift-machine-api
+----
++
+Or: 
++
+[source,terminal]
+----
+$ oc edit machineset <machineset> -n openshift-machine-api
+----
+For more information on scaling machine sets, see "Modifying a compute machine set".
+
++
+--
+.Example `MachineSet` with a taint set
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata: [...]
+spec:
+  [...]
+  template:
+    [...]
+    spec:
+      [...]
+      taints:
+      - effect: NoSchedule         
+        key: multi-arch.openshift.io/arch                
+        value: arm64
+----
+You can also set a taint on a specific node by running the following command: 
+[source,terminal]
+----
+$ oc adm taint nodes <node-name> multi-arch.openshift.io/arch=arm64:NoSchedule
+----
+--
+
+Creating a default toleration:: You can annotate a namespace so all of the workloads get the same default toleration by running the following command:
++
+[source,terminal]
+----
+$ oc annotate namespace my-namespace \
+  'scheduler.alpha.kubernetes.io/defaultTolerations'='[{"operator": "Exists", "effect": "NoSchedule", "key": "multi-arch.openshift.io/arch"}]'
+----
+
+Tolerating in workloads when the architecture taint is set:: On a node with a defined taint, workloads will not be scheduled on that node. However, you can allow them to be scheduled by setting a toleration in the pod specification.
```

Comment:
> Tolerating in workloads when the architecture taint is set

this title seems missing an object: "tolerating architecture taints in workloads"?